src/lib/blog/utils.test.ts

```diff
@@ -0,0 +1,13 @@
+import { test, expect } from 'vitest';
+import { readingTime } from './utils';
+import fs from 'fs';
+
+test('properly times plain text strings', () => {
+  const string = fs.readFileSync('src/lib/blog/bee-movie.txt').toString();
```

Comment:
Nit: I would avoid calling the name of the variable the same thing as a primitive type. The variable named `string` could instead be named `sampleX` or `exampleX` where `X` is `Content`, `Corpus`, `Text`, etc. This occurs on lines 6, 7, 11, and 12. Similar: <https://github.com/EthanThatOneKid/acmcsuf.com/pull/481#discussion_r882222338>.

Reply:
MAN

Reply:
maybe just get good jason ez